applications/ardas.group.inc/squad.md

```diff
@@ -0,0 +1,39 @@
+# Ardas Group Inc.
+
+https://ardas-it.com/
+
+Склад команди:
+
+`12` [**Болонський** Олексій](/players/bolonskiy.oleksiy.19860522.jpg)	`/1986-05-22/`
+
+`8`	 [**Борисов** Ігорь](/players/borysov.igor.19950605.jpg)	`/1995-06-05/`
```

Comment:
Please use a single whitespace symbol (instead of tabulation) as a separator between number, name, birth date and company name

Reply:
@osaula9 Still an issue. Tabulations are present